### PR TITLE
Feat: added config files for testing 

### PR DESCRIPTION
**Pull request by sempitrnal**

### CODE REVIEW

File: WebApp/frontend/quiz-master/__tests__/Home.test.tsx
Comment (jaymar921) [inline WebApp/frontend/quiz-master/__tests__/Home.test.tsx:1]: nice, a template, this could be useful

File: WebApp/frontend/quiz-master/jest.config.js
Comment (jaymar921) [inline WebApp/frontend/quiz-master/jest.config.js:1]: all is good 🚀

**Review by jaymar921 — APPROVED** [review]
approved 🚀

**Review by JMark-FS — APPROVED** [review]
looks good

**Pull request merged**